### PR TITLE
All props needs to follow the same format

### PR DESCRIPTION
**Pull request by cableman**
#### Link to issue

https://platform.dandigbib.org/issues/5387

#### Description

When using dpl-react in the CMS the props need to follow this format.

#### Screenshot of the result

N/A

#### Checklist

- [ ] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [ ] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [ ] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

#### Additional comments or questions

N/A

### CODE REVIEW

File: src/apps/loan-list/helpers.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import get from "lodash.get";
 import dayjs from "dayjs";
-import { LoanV2 } from "../../core/fbs/model/loanV2";
+import { LoanV2 } from "../../core/fbs/model";
```
Comment (sinejespersen) [inline src/apps/loan-list/helpers.ts:4]: why this change

Comment (cableman) [reply src/apps/loan-list/helpers.ts:4]: The IDE that i uses says that this is the correct way to import it...

**Review by sinejespersen — APPROVED** [review]
